lib/cabbage/missing_step_advisor.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Cabbage.MissingStepAdvisor do
+  @moduledoc """
+  Raises an error, because a feature step is missing its implementation.
+
+  The message of the error will give the user a useful code snippet where
+  variables in feature steps are converted to regex capture groups.
+  """
+
+  def raise(step_text, step_type) do
+
+    converted_step_text =
+      step_text
+      |> convert_numbers()
+      |> convert_double_quote_strings()
+      |> convert_single_quote_strings()
+
+    raise """
+    Please add a matching step for:
+    "#{step_type} #{step_text}"
+
+      def#{step_type |> String.downcase} ~r/^#{converted_step_text}$/, vars, state do
+        # Your implementation here
+      end
+    """
+  end
+
+  defp convert_numbers(step_text) do
+    Regex.replace(~r/(\s)\d+(\s|$)/, step_text, ~s/\\1(?<number>\\d+)\\2/)
```

Comment:
you were right, made the change https://github.com/cabbage-ex/cabbage/pull/14/commits/1ffed9f0d32a8032012e511e832fa0e50aac923f